pkg/controller/useraccountstatus/useraccount_status_controller.go

```diff
@@ -114,6 +114,7 @@ func (r *ReconcileUserAccountStatus) updateMasterUserRecord(userAcc *toolchainv1
 	for i, account := range mur.Spec.UserAccounts {
 		if account.TargetCluster == fedCluster.OwnerClusterName {
 			mur.Spec.UserAccounts[i].SyncIndex = userAcc.ResourceVersion
+			mur.Spec.UserAccounts[i].Spec.Disabled = userAcc.Spec.Disabled
```

Comment:
Why would we need that? We are not supposed to change MUR Spec from member/UserAccount. It works the other way around.
`SyncIndex` is a special case. It's just a way for us to trigger a MUR reconcile when UserAccount has changed.